EventKit/eventkit-cloud
# Add favorite UI

# Description of Improvement

Adds favorite icon to provider list in Create workflow step two. 
Sets up logic to update favorite backend when favorite icon is pressed.
Updates tests to verify appropriate rendering with favorite icon

## How to test

Verify favorite icon appears
Verify favorite can be toggled, and maintains expected state on refresh
Verify tests pass

## Quality Assurance

The following items have been updated:


## Screenshots

<img width="1786" alt="Screen Shot 2022-08-31 at 5 19 09 PM" src="https://user-images.githubusercontent.com/3164631/187788063-6aafd460-0bef-462e-b59a-0d555debb9b4.png">


:smile:
